mcs/class/referencesource/mscorlib/system/security/cryptography/rc2cryptoserviceprovider.cs

```diff
@@ -63,7 +63,12 @@ public sealed class RC2CryptoServiceProvider : RC2 {
                 return m_use40bitSalt;
             }
             set {
+#if MONO
```

Comment:
Updated. I decided to keep NotImplementedException since there's technically nothing stopping us from supporting it one day.